pkgs/development/libraries/gstreamer/bad/default.nix

```diff
@@ -85,6 +137,38 @@ stdenv.mkDerivation rec {
     # TODO: mjpegtools uint64_t is not compatible with guint64 on Darwin
     ++ optional (!stdenv.isDarwin) mjpegtools;
 
+  mesonFlags = [
+    # Enables all features, so that we know when new dependencies are necessary.
+    "-Dauto_features=enabled"
+
+    "-Dexamples=disabled" # requires many dependencies and probably not useful for our users
+
+    "-Ddts=disabled" # required `libdca` library not packaged in nixpkgs as of writing, and marked as "BIG FAT WARNING: libdca is still in early development"
+    "-Dfaac=${if faacSupport then "enabled" else "disabled"}"
+    "-Dgsm=disabled" # as of writing, with `gsm` in `buildInputs` we get "GSM plugin is enabled: found libgsm but no headers"; gsm packaging problem?
```

Comment:
Looks like a typo, there is also incorrect indentation: https://gitlab.freedesktop.org/gstreamer/gst-plugins-bad/blob/3a0a82a783b2a8af4c0962bcf3b238ca927e4714/ext/gsm/meson.build#L8

Reply:
Yes looks like it.

Even when fixing it (see my [upstream PR](https://gitlab.freedesktop.org/gstreamer/gst-plugins-bad/merge_requests/312) I made based on your comment), it still fails to compile because of:

https://gitlab.freedesktop.org/gstreamer/gst-plugins-bad/merge_requests/312#note_155041

Reply:
Fixed that too now: https://gitlab.freedesktop.org/gstreamer/gst-plugins-bad/merge_requests/312#note_155052

Reply:
Done